source/develop/developer-guide/vdsm/developers.html.md

```diff
@@ -18,12 +18,24 @@ wiki_last_updated: 2015-08-26
 To build VDSM, enable the oVirt repositories by installing the ovirt-release rpm:
 
       yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
-
+      
+      
+ or if you are using Fedora 
+ 
+ 
+      dnf install http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
```

Comment:
I think yes The casual admin may find it hard especially the new contributors .Else you decide?